lightly/models/modules/masked_autoencoder_timm.py

```diff
@@ -5,9 +5,10 @@
 import torch.nn as nn
 from timm.models.vision_transformer import Block
 from torch import Tensor
-from torch.nn import LayerNorm, Linear, Module, Parameter, Sequential
+from torch.nn import LayerNorm, Module, Parameter, Sequential
 
 from lightly.models import utils
+from lightly.models.modules.masked_vision_transformer_timm import _init_weights
```

Comment:
Functions starting with an underscore are considered "private" and should not be accessed from outside the module/file. Could you rename the function in `masked_vision_transformer_timm.py` to `init_weights` instead to make it public?